src/lib/spinel/spinel.h

```diff
@@ -4814,6 +4803,41 @@ enum
 
     SPINEL_PROP_SRP_SERVER__END = 0x930,
 
+    SPINEL_PROP_DNSSD__BEGIN = 0x930,
+
+    /// Dnssd State
+    /** Format `b`: Write-only
+     *
+     * `C`: The dnssd state.
+     */
+    SPINEL_PROP_DNSSD_STATE = SPINEL_PROP_DNSSD__BEGIN + 1,
+
+    /// Dnssd Request Result
+    /** Format `LC`: Write
+     *
+     * `L` : The Dnssd Request ID.
+     * `C` : The result of the request. A unsigned int8 corresponds to otError.
+     * `D` : The context of the request. (A pointer to the callback for the request)
+     *
+     * Host uses this property to notify the NCP of the result of NCP's DNS-SD request.
+     */
+    SPINEL_PROP_DNSSD_REQUEST_RESULT = SPINEL_PROP_DNSSD__BEGIN + 2,
+
+    /// DNS-SD Host
+    /** Format `LUSA(6)`: Inserted/Removed
+     *
+     * `L`    : The Dnssd Request ID.
+     * `U`    : The host name.
+     * `S`    : The count of IPv6 addresses.
+     * `A(6)` : The IPv6 addresses of the host.
+     * `D`    : The context of the request. (A pointer to the callback for the request)
```

Comment:
From previous suggestion, we can re-order as:
```suggestion
     * `U`    : The host name.
     * `S`    : The count of IPv6 addresses.
     * `A(6)` : The IPv6 addresses of the host.
     * `L`    : The Dnssd Request ID.
     * `D`    : The context of the request. (A pointer to the callback for the request)
```